src/components/CaptureMatching/CaptureMatchingView.js

```diff
@@ -427,8 +426,8 @@ function CaptureMatchingView() {
   const getGroverRegDate = () => {
     return growerAccount.first_registration_at
       ? 'Joined at ' +
-          format(new Date(growerAccount.first_registration_at), 'MM/dd/yyyy')
-      : 'Joined at ' + format(new Date(growerAccount.created_at), 'MM/dd/yyyy');
+          getDateFormatted(growerAccount.first_registration_at, 'dd.MM.yyyy')
```

Comment:
The intention is to format this date according to the user's locale-language (e.g., `mm/dd/yyyy` for `en-US`, `yyyy/mm/dd` for `zh-ZH`) using [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat).

`getDateTimeStringLocale()` already does the formatting we want, so it'd be better to use or update that than create a new method.